From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Mon, 29 Aug 2016 21:19:32 +0200
Subject: Fix various typos in ChangeLog

This file isn't present in the upstream VCS, but generated from the Git
commit messages during the release process.  Thus fixing typos in it
isn't upstreamable.
---
 ChangeLog | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 824141d..17b930c 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -9617,7 +9617,7 @@
 
 2012-02-07  Fabio M. Di Nitto  <fdinitto@redhat.com>
 
-	votequorum: fix quorum_ringid setting before any delivery occours
+	votequorum: fix quorum_ringid setting before any delivery occurs
 	Reviewed-by: Christine Caulfield <ccaulfie@redhat.com>
 
 2012-02-07  Angus Salkeld  <asalkeld@redhat.com>
@@ -9802,7 +9802,7 @@
 
 	Honor exec_init_fn call
 	exec_init_fn now either returns NULL (success) or a string which indicates
-	the error that occured during service engine initialization.  If an error
+	the error that occurred during service engine initialization.  If an error
 	occurs, corosync will exit.  This patch adds ykd and makes other suggestions
 	from Fabio Di Nitto.
 
@@ -11166,7 +11166,7 @@
 
 	start 5 nodes randomly at about same time, start 5 nodes randomly at about
 	same time, wait 10 seconds and attempt to send a message.  If message blocks
-	on "TRY_AGAIN" likely a message loss has occured.  Wait a few minutes without
+	on "TRY_AGAIN" likely a message loss has occurred.  Wait a few minutes without
 	cyclng the nodes and see if the TRY_AGAIN state becomes unblocked.
 
 	If it doesn't the test case has failed
@@ -11557,7 +11557,7 @@
 	is these last messages should be delivered after a transitional
 	configuration to maintain SAFE agreement.  We don't have support for
 	SAFE atm, so it is probably safe just to throw these messages away.  Without
-	this change, the new message queue on a new configuraton change is out of sync.
+	this change, the new message queue on a new configuration change is out of sync.
 
 	Tested-by: Tim Beale <tlbeale@gmail.com>
 	Reviewed-by: Jan Friesse <jfriesse@redhat.com>
@@ -19305,7 +19305,7 @@
 
 2007-08-11  Steven Dake  <sdake@redhat.com>
 
-	Patch to not overwrite defualt configuration file or amf.conf file when executing make install
+	Patch to not overwrite default configuration file or amf.conf file when executing make install
 	git-svn-id: http://svn.fedorahosted.org/svn/corosync/trunk@1420 fd59a12c-fef9-0310-b244-a6a79926bd2f
 
 	Allow make doxygen to work properly
@@ -19648,7 +19648,7 @@
 
 2006-11-10  Steven Dake  <sdake@redhat.com>
 
-	Update checkpoint service from all qualifications that have occured in whitetank branch.
+	Update checkpoint service from all qualifications that have occurred in whitetank branch.
 	git-svn-id: http://svn.fedorahosted.org/svn/corosync/trunk@1299 fd59a12c-fef9-0310-b244-a6a79926bd2f
 
 	Recovery algorithms described in this document.
@@ -25822,13 +25822,13 @@
 
 2004-07-12  Steven Dake  <sdake@broked.org>
 
-	Retransmit orf token if no multicast or token received within a timeout period.  This could mean that the token was lost, but no configuration change has really occured.  Rather then execute the heavyweight membership protocol, just retry the token and if it fails after the token timeout, execute a configuration change sequence.
+	Retransmit orf token if no multicast or token received within a timeout period.  This could mean that the token was lost, but no configuration change has really occurred.  Rather then execute the heavyweight membership protocol, just retry the token and if it fails after the token timeout, execute a configuration change sequence.
 	BKrev: 40f21082dvUxH0aOPvKmpqoqlkEgEA
 
 
 	git-svn-id: http://svn.fedorahosted.org/svn/corosync/trunk@99 fd59a12c-fef9-0310-b244-a6a79926bd2f
 
-	Add retransmit of token if no multicast or token received within a timeout period (100 msec).  This helps avoid a reconfiguration when only the token is lost, but no real configuration changes have occured.
+	Add retransmit of token if no multicast or token received within a timeout period (100 msec).  This helps avoid a reconfiguration when only the token is lost, but no real configuration changes have occurred.
 	(Logical change 1.32)
 
 
